doc/index.rst

```diff
@@ -0,0 +1,176 @@
+.. acis_thermal_check documentation master file
+
+ACIS Thermal Model Tools
+========================
+
+This suite provides the tools to use and maintain the Chandra ACIS
+thermal models. The key elements are:
+
+  - ``dea_check``: Thermal check of command loads and validate DEA
+    model against recent telemetry
+  - ``dpa_check``: Thermal check of command loads and validate DPA
+    model against recent telemetry
+  - ``psmc_check``: Thermal check of command loads and validate PSMC
+    model against recent telemetry
+
+These tools depend on Sybase tables and in particular the commanded states database
+which is accessed primarily via the Chandra.cmd_states_ module.
+
+.. _Chandra.cmd_states: http://cxc.harvard.edu/mta/ASPECT/tool_doc/cmd_states/
+
+``acis_thermal_check``
+======================
+
+Overview
+--------
+
+This code generates backstop load review outputs for checking the ACIS temperatures
+1DEAMZT, 1DPAMZT, and 1PDEAAT. It also generates model validation plots for these
+temperatures comparing predicted values to telemetry for the previous three weeks.
+
+Installation and Development
+----------------------------
+
+This assumes that you have a cloned copy of ``acis_thermal_check`` from
+http://github.com/acisops/acis_thermal_check. If you have your own Python/Ska
+environment available (on a laptop or some other machine), and you have write
+permissions on it, to install the package simply run:
+
+::
+
+    python setup.py install
+
+from the top-level directory of the package. This will install the three scripts
+``dea_check``, ``dpa_check``, and ``psmc_check`` into your path, and install
+``acis_thermal_check`` as a Python package, which can then be imported into any Python
+script using the same ``python`` executable.
+
+If you are running and testing this package using the flight-approved Ska environment
+on the HEAD LAN, you will not be able to directly install into this environment but
+can install ``acis_thermal_check`` as a local package. In this case, the command
+is slightly modified:
+
+::
+
+    python setup.py install --user
```

Comment:
I feel like `--user` tends to cause more problems than  it solves because the installation is not environment-specific.  I also have shot myself in the foot numerous times by forgetting about a package that was installed in user.  Having said that, I understand the motivation here and it's fine to leave this for now.  I hope it will not be too long before we have all of Ska available as conda packages and it will be a snap to create a dev environment and install there.  This is a high priority.